src/NotificationBar/NotificationBar.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { StyledNotificationBar } from './NotificationBar.styles';
+
+export type Props = {
+  children?: React.ReactElement;
+  variant?: 'alert' | 'info';
+};
+
+export const NotificationBar: React.FC<Props> = props => {
```

Comment:
Can this default the variant to 'info', checkout Icons.tsx for defaulting pros in typescript